### PR TITLE
Feature/pg

### PR DESCRIPTION
**Pull request by aiharisov**

### CODE REVIEW

File: workout_portal/serializers/training_serializer.py [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+class TrainingSerializer(serializers.ModelSerializer):
+    user = UserSerializer()
+    exercises = ExerciseSerializer(many=True, read_only=True)
+
     class Meta:
         model = Training
-        fields = ('name', 'date', 'user')
+        fields = ('name', 'date', 'user', 'exercises')
+
+    # def to_representation(self, value):
```
Comment (yazabara) [inline workout_portal/serializers/training_serializer.py]: Could you remove unsed code?

File: workout_portal/service/training_service.py [hunk 2]
```diff
@@ -7,8 +7,9 @@ def create_new(user_id, name, date):
         return Training.objects.create(user_id=user_id, name=name, date=date)
 
     @staticmethod
-    def list(limit=5):
-        return Training.objects.order_by('date')[0:limit]
+    def list():
+        trainings = Training.objects.all()
+        return trainings
```
Comment (yazabara) [inline workout_portal/service/training_service.py]: return `Training.objects.all()` or `Training.objects.order_by('date')`

File: workout_portal/urls.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
-# from . import views
+from . import views
 
 
 urlpatterns = [
-    # path('', controller.index, name='index'),
-    # path('training/add', controller.add_training, name='add_training'),
-    # path('training/<int:training_id>/', controller.get_training, name='get_training'),
+    path('', controller.index, name='index'),
```
Comment (yazabara) [inline workout_portal/urls.py]: why you uncommented this code? This endpoints already defined in `django_learning/urls.py` file with ViewSets

File: workout_portal/serializers/training_serializer.py [hunk 4]
```diff
@@ -1,9 +1,18 @@
...
 
-class TrainingSerializer(serializers.HyperlinkedModelSerializer):
+class TrainingSerializer(serializers.ModelSerializer):
+    user = UserSerializer()
+    exercises = ExerciseSerializer(many=True, read_only=True)
+
     class Meta:
         model = Training
-        fields = ('name', 'date', 'user')
+        fields = ('name', 'date', 'user', 'exercises')
```
Comment (ichurakov) [inline workout_portal/serializers/training_serializer.py]: i think "id" field will need on front.. For example, for adding review.

**Pull request closed without merging**